.github/workflows/release.yml

```diff
@@ -195,6 +195,16 @@ jobs:
           go-version: ">=1.23.8"
           cache-dependency-path: "**/go.sum"
 
+      - name: Setup Rust
+        uses: actions-rs/toolchain@v1
```

Comment:
This action is archived.
I've seen some repositories using [this](https://github.com/dtolnay/rust-toolchain) one.